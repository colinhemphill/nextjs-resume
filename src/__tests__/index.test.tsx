import React from 'react';
import {
  educationalExperiences as prismicEducationalExperiences,
  links as prismicLinks,
  personalInformation as prismicPersonalInformation,
  professionalExperiences as prismicProfessionalExperiences,
  skills as prismicSkills,
} from '../fixtures/prismic';
import Index from '../pages/index';
import { render } from '../testing/render';

const DEFAULT_ENV = process.env;

describe('Render the main page', () => {
  beforeEach(() => {
    jest.resetModules();
    process.env = { ...DEFAULT_ENV };
  });

  afterAll(() => {
    process.env = DEFAULT_ENV;
  });

  it('Uses a Prismic integration', () => {
    process.env.NEXT_PUBLIC_CMS_INTEGRATION = 'prismic';
    const { container } = render(
      <Index
        educationalExperiences={prismicEducationalExperiences}
        links={prismicLinks}
        personalInformation={prismicPersonalInformation}
        professionalExperiences={prismicProfessionalExperiences}
        skills={prismicSkills}
      />,
    );
    expect(container).toMatchSnapshot();
  });
});
