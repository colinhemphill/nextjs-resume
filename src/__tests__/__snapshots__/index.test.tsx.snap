// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render the main page Uses a Prismic integration 1`] = `
<div>
  <header
    class="header"
  >
    <div
      class="container"
    >
      <div
        class="row align-items-center text-md-start text-center"
      >
        <div
          class="col-md"
        >
          <h1
            class="text-white"
          >
            Some Body
          </h1>
          <h2
            class="text-white mb-0"
          >
            Cool Person
          </h2>
        </div>
        <div
          class="col-md-auto mt-md-0 mt-xxs d-print-none d-block"
        >
          <a
            class="btn btn-dark btn-lg"
            href="/api/pdf"
            rel="noopener noreferrer"
            target="_blank"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-file-pdf me-xxxs"
              data-icon="file-pdf"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 384 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M184 208c0-4.406-3.594-8-8-8S168 203.6 168 208c0 2.062 .2969 23.31 9.141 50.25C179.1 249.6 184 226.2 184 208zM256 0v128h128L256 0zM80 422.4c0 9.656 10.47 11.97 14.38 6.375C99.27 421.9 108.8 408 120.1 388.6c-14.22 7.969-27.25 17.31-38.02 28.31C80.75 418.3 80 420.3 80 422.4zM224 128L224 0H48C21.49 0 0 21.49 0 48v416C0 490.5 21.49 512 48 512h288c26.51 0 48-21.49 48-48V160h-127.1C238.3 160 224 145.7 224 128zM292 312c24.26 0 44 19.74 44 44c0 24.67-18.94 44-43.13 44c-5.994 0-11.81-.9531-17.22-2.805c-20.06-6.758-38.38-15.96-54.55-27.39c-23.88 5.109-45.46 11.52-64.31 19.1c-14.43 26.31-27.63 46.15-36.37 58.41C112.1 457.8 100.8 464 87.94 464C65.92 464 48 446.1 48 424.1c0-11.92 3.74-21.82 11.18-29.51c16.18-16.52 37.37-30.99 63.02-43.05c11.75-22.83 21.94-46.04 30.33-69.14C136.2 242.4 136 208.4 136 208c0-22.05 17.95-40 40-40c22.06 0 40 17.95 40 40c0 24.1-7.227 55.75-8.938 62.63c-1.006 3.273-2.035 6.516-3.082 9.723c7.83 14.46 17.7 27.21 29.44 38.05C263.1 313.4 284.3 312.1 287.6 312H292zM156.5 354.6c17.98-6.5 36.13-11.44 52.92-15.19c-12.42-12.06-22.17-25.12-29.8-38.16C172.3 320.6 164.4 338.5 156.5 354.6zM292.9 368C299 368 304 363 304 356.9C304 349.4 298.6 344 292 344H288c-.3438 .0313-16.83 .9687-40.95 4.75c11.27 7 24.12 13.19 38.84 18.12C288 367.6 290.5 368 292.9 368z"
                fill="currentColor"
              />
            </svg>
            Download as PDF
          </a>
        </div>
      </div>
    </div>
  </header>
  <section
    class="bg-white py-sm"
  >
    <div
      class="container"
    >
      <div
        class="row"
      >
        <div
          class="col-md"
        >
          <article>
            <h3>
              <span
                class="fa-layers fa-fw me-xxxs"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-circle "
                  data-icon="circle"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M512 256C512 397.4 397.4 512 256 512C114.6 512 0 397.4 0 256C0 114.6 114.6 0 256 0C397.4 0 512 114.6 512 256z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-user "
                  color="white"
                  data-icon="user"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style="transform-origin: 0.4375em 0.5em;"
                  viewBox="0 0 448 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    transform="translate(224 256)"
                  >
                    <g
                      transform="translate(0, 0)  scale(0.5, 0.5)  rotate(0 0 0)"
                    >
                      <path
                        d="M224 256c70.7 0 128-57.31 128-128s-57.3-128-128-128C153.3 0 96 57.31 96 128S153.3 256 224 256zM274.7 304H173.3C77.61 304 0 381.6 0 477.3c0 19.14 15.52 34.67 34.66 34.67h378.7C432.5 512 448 496.5 448 477.3C448 381.6 370.4 304 274.7 304z"
                        fill="currentColor"
                        transform="translate(-224 -256)"
                      />
                    </g>
                  </g>
                </svg>
              </span>
              About Me
            </h3>
            <p>
              An About Me description.
            </p>
          </article>
        </div>
        <div
          class="col-md mt-md-0 mt-xs"
        >
          <article>
            <h3>
              <span
                class="fa-layers fa-fw me-xxxs"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-circle "
                  data-icon="circle"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M512 256C512 397.4 397.4 512 256 512C114.6 512 0 397.4 0 256C0 114.6 114.6 0 256 0C397.4 0 512 114.6 512 256z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-id-card "
                  color="white"
                  data-icon="id-card"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style="transform-origin: 0.5625em 0.5em;"
                  viewBox="0 0 576 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    transform="translate(288 256)"
                  >
                    <g
                      transform="translate(0, 0)  scale(0.5, 0.5)  rotate(0 0 0)"
                    >
                      <path
                        d="M528 32h-480C21.49 32 0 53.49 0 80V96h576V80C576 53.49 554.5 32 528 32zM0 432C0 458.5 21.49 480 48 480h480c26.51 0 48-21.49 48-48V128H0V432zM368 192h128C504.8 192 512 199.2 512 208S504.8 224 496 224h-128C359.2 224 352 216.8 352 208S359.2 192 368 192zM368 256h128C504.8 256 512 263.2 512 272S504.8 288 496 288h-128C359.2 288 352 280.8 352 272S359.2 256 368 256zM368 320h128c8.836 0 16 7.164 16 16S504.8 352 496 352h-128c-8.836 0-16-7.164-16-16S359.2 320 368 320zM176 192c35.35 0 64 28.66 64 64s-28.65 64-64 64s-64-28.66-64-64S140.7 192 176 192zM112 352h128c26.51 0 48 21.49 48 48c0 8.836-7.164 16-16 16h-192C71.16 416 64 408.8 64 400C64 373.5 85.49 352 112 352z"
                        fill="currentColor"
                        transform="translate(-288 -256)"
                      />
                    </g>
                  </g>
                </svg>
              </span>
              Contact Information
            </h3>
            <ul
              class="list-unstyled"
            >
              <li>
                <div
                  class="row mt-md-0 mt-xxxs"
                >
                  <div
                    class="col-lg-2 col-sm-3"
                  >
                    <strong>
                      Location:
                    </strong>
                  </div>
                  <div
                    class="col"
                  >
                    Somewhere Only We Know
                  </div>
                </div>
              </li>
            </ul>
          </article>
        </div>
      </div>
      <article
        class="mt-xs"
      >
        <h3>
          <span
            class="fa-layers fa-fw me-xxxs"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-circle "
              data-icon="circle"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M512 256C512 397.4 397.4 512 256 512C114.6 512 0 397.4 0 256C0 114.6 114.6 0 256 0C397.4 0 512 114.6 512 256z"
                fill="currentColor"
              />
            </svg>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-check "
              color="white"
              data-icon="check"
              data-prefix="fas"
              focusable="false"
              role="img"
              style="transform-origin: 0.4375em 0.5em;"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                transform="translate(224 256)"
              >
                <g
                  transform="translate(0, 0)  scale(0.5, 0.5)  rotate(0 0 0)"
                >
                  <path
                    d="M438.6 105.4C451.1 117.9 451.1 138.1 438.6 150.6L182.6 406.6C170.1 419.1 149.9 419.1 137.4 406.6L9.372 278.6C-3.124 266.1-3.124 245.9 9.372 233.4C21.87 220.9 42.13 220.9 54.63 233.4L159.1 338.7L393.4 105.4C405.9 92.88 426.1 92.88 438.6 105.4H438.6z"
                    fill="currentColor"
                    transform="translate(-224 -256)"
                  />
                </g>
              </g>
            </svg>
          </span>
          Skills & Expertise
        </h3>
        <div
          class="row mt-xxs"
        >
          <div
            class="col-lg"
          >
            <h4
              class="h5 text-uppercase"
            >
              <span
                class="text-primary"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-star "
                  data-icon="star"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 576 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M381.2 150.3L524.9 171.5C536.8 173.2 546.8 181.6 550.6 193.1C554.4 204.7 551.3 217.3 542.7 225.9L438.5 328.1L463.1 474.7C465.1 486.7 460.2 498.9 450.2 506C440.3 513.1 427.2 514 416.5 508.3L288.1 439.8L159.8 508.3C149 514 135.9 513.1 126 506C116.1 498.9 111.1 486.7 113.2 474.7L137.8 328.1L33.58 225.9C24.97 217.3 21.91 204.7 25.69 193.1C29.46 181.6 39.43 173.2 51.42 171.5L195 150.3L259.4 17.97C264.7 6.954 275.9-.0391 288.1-.0391C300.4-.0391 311.6 6.954 316.9 17.97L381.2 150.3z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-star secondStar"
                  data-icon="star"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 576 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M381.2 150.3L524.9 171.5C536.8 173.2 546.8 181.6 550.6 193.1C554.4 204.7 551.3 217.3 542.7 225.9L438.5 328.1L463.1 474.7C465.1 486.7 460.2 498.9 450.2 506C440.3 513.1 427.2 514 416.5 508.3L288.1 439.8L159.8 508.3C149 514 135.9 513.1 126 506C116.1 498.9 111.1 486.7 113.2 474.7L137.8 328.1L33.58 225.9C24.97 217.3 21.91 204.7 25.69 193.1C29.46 181.6 39.43 173.2 51.42 171.5L195 150.3L259.4 17.97C264.7 6.954 275.9-.0391 288.1-.0391C300.4-.0391 311.6 6.954 316.9 17.97L381.2 150.3z"
                    fill="currentColor"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-star thirdStar"
                  data-icon="star"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 576 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M381.2 150.3L524.9 171.5C536.8 173.2 546.8 181.6 550.6 193.1C554.4 204.7 551.3 217.3 542.7 225.9L438.5 328.1L463.1 474.7C465.1 486.7 460.2 498.9 450.2 506C440.3 513.1 427.2 514 416.5 508.3L288.1 439.8L159.8 508.3C149 514 135.9 513.1 126 506C116.1 498.9 111.1 486.7 113.2 474.7L137.8 328.1L33.58 225.9C24.97 217.3 21.91 204.7 25.69 193.1C29.46 181.6 39.43 173.2 51.42 171.5L195 150.3L259.4 17.97C264.7 6.954 275.9-.0391 288.1-.0391C300.4-.0391 311.6 6.954 316.9 17.97L381.2 150.3z"
                    fill="currentColor"
                  />
                </svg>
              </span>
               
              Proficient
            </h4>
            <p>
              JavaScript ES6, TypeScript, Test Writing, Making Stuff Up
            </p>
          </div>
        </div>
      </article>
    </div>
  </section>
  <section
    class="bg-light py-sm"
  >
    <div
      class="container"
    >
      <h3>
        <span
          class="fa-layers fa-fw me-xxxs"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-circle "
            data-icon="circle"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M512 256C512 397.4 397.4 512 256 512C114.6 512 0 397.4 0 256C0 114.6 114.6 0 256 0C397.4 0 512 114.6 512 256z"
              fill="currentColor"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-briefcase "
            color="white"
            data-icon="briefcase"
            data-prefix="fas"
            focusable="false"
            role="img"
            style="transform-origin: 0.5em 0.5em;"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              transform="translate(256 256)"
            >
              <g
                transform="translate(0, 0)  scale(0.5, 0.5)  rotate(0 0 0)"
              >
                <path
                  d="M320 336c0 8.844-7.156 16-16 16h-96C199.2 352 192 344.8 192 336V288H0v144C0 457.6 22.41 480 48 480h416c25.59 0 48-22.41 48-48V288h-192V336zM464 96H384V48C384 22.41 361.6 0 336 0h-160C150.4 0 128 22.41 128 48V96H48C22.41 96 0 118.4 0 144V256h512V144C512 118.4 489.6 96 464 96zM336 96h-160V48h160V96z"
                  fill="currentColor"
                  transform="translate(-256 -256)"
                />
              </g>
            </g>
          </svg>
        </span>
        Professional Experience
      </h3>
      <article
        class="mt-xs"
      >
        <h4
          class="d-inline-block"
        >
          <span
            class="bg-dark px-xxxs text-white rounded"
          >
            My Title
          </span>
           
          at 
          The Zebra
        </h4>
        <div>
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-calendar me-xxxs"
            data-icon="calendar"
            data-prefix="far"
            focusable="false"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M152 64H296V24C296 10.75 306.7 0 320 0C333.3 0 344 10.75 344 24V64H384C419.3 64 448 92.65 448 128V448C448 483.3 419.3 512 384 512H64C28.65 512 0 483.3 0 448V128C0 92.65 28.65 64 64 64H104V24C104 10.75 114.7 0 128 0C141.3 0 152 10.75 152 24V64zM48 448C48 456.8 55.16 464 64 464H384C392.8 464 400 456.8 400 448V192H48V448z"
              fill="currentColor"
            />
          </svg>
          February 2018
          –
          Current
        </div>
        <div
          class="mt-xxs"
        >
          <p>
            A description of my role
          </p>
        </div>
      </article>
    </div>
  </section>
  <section
    class="bg-white py-sm"
  >
    <div
      class="container"
    >
      <h3>
        <span
          class="fa-layers fa-fw me-xxxs"
        >
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-circle "
            data-icon="circle"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M512 256C512 397.4 397.4 512 256 512C114.6 512 0 397.4 0 256C0 114.6 114.6 0 256 0C397.4 0 512 114.6 512 256z"
              fill="currentColor"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-graduation-cap "
            color="white"
            data-icon="graduation-cap"
            data-prefix="fas"
            focusable="false"
            role="img"
            style="transform-origin: 0.625em 0.5em;"
            viewBox="0 0 640 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              transform="translate(320 256)"
            >
              <g
                transform="translate(0, 0)  scale(0.5, 0.5)  rotate(0 0 0)"
              >
                <path
                  d="M623.1 136.9l-282.7-101.2c-13.73-4.91-28.7-4.91-42.43 0L16.05 136.9C6.438 140.4 0 149.6 0 160s6.438 19.65 16.05 23.09L76.07 204.6c-11.89 15.8-20.26 34.16-24.55 53.95C40.05 263.4 32 274.8 32 288c0 9.953 4.814 18.49 11.94 24.36l-24.83 149C17.48 471.1 25 480 34.89 480H93.11c9.887 0 17.41-8.879 15.78-18.63l-24.83-149C91.19 306.5 96 297.1 96 288c0-10.29-5.174-19.03-12.72-24.89c4.252-17.76 12.88-33.82 24.94-47.03l190.6 68.23c13.73 4.91 28.7 4.91 42.43 0l282.7-101.2C633.6 179.6 640 170.4 640 160S633.6 140.4 623.1 136.9zM351.1 314.4C341.7 318.1 330.9 320 320 320c-10.92 0-21.69-1.867-32-5.555L142.8 262.5L128 405.3C128 446.6 213.1 480 320 480c105.1 0 192-33.4 192-74.67l-14.78-142.9L351.1 314.4z"
                  fill="currentColor"
                  transform="translate(-320 -256)"
                />
              </g>
            </g>
          </svg>
        </span>
        Education
      </h3>
      <article
        class="mt-xs"
      >
        <h4
          class="d-inline-block"
        >
          B.S. in Cool Person
        </h4>
        <div>
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-building-columns me-xxxs"
            data-icon="building-columns"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M243.4 2.587C251.4-.8625 260.6-.8625 268.6 2.587L492.6 98.59C506.6 104.6 514.4 119.6 511.3 134.4C508.3 149.3 495.2 159.1 479.1 160V168C479.1 181.3 469.3 192 455.1 192H55.1C42.74 192 31.1 181.3 31.1 168V160C16.81 159.1 3.708 149.3 .6528 134.4C-2.402 119.6 5.429 104.6 19.39 98.59L243.4 2.587zM256 128C273.7 128 288 113.7 288 96C288 78.33 273.7 64 256 64C238.3 64 224 78.33 224 96C224 113.7 238.3 128 256 128zM127.1 416H167.1V224H231.1V416H280V224H344V416H384V224H448V420.3C448.6 420.6 449.2 420.1 449.8 421.4L497.8 453.4C509.5 461.2 514.7 475.8 510.6 489.3C506.5 502.8 494.1 512 480 512H31.1C17.9 512 5.458 502.8 1.372 489.3C-2.715 475.8 2.515 461.2 14.25 453.4L62.25 421.4C62.82 420.1 63.41 420.6 63.1 420.3V224H127.1V416z"
              fill="currentColor"
            />
          </svg>
          Cool U
        </div>
        <div
          class="mt-xxs"
        >
          <p>
            A description of my education
          </p>
        </div>
      </article>
    </div>
  </section>
  <section
    class="bg-light py-sm"
  >
    <div
      class="container"
    >
      <article>
        <h3>
          <span
            class="fa-layers fa-fw me-xxxs"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-circle "
              data-icon="circle"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M512 256C512 397.4 397.4 512 256 512C114.6 512 0 397.4 0 256C0 114.6 114.6 0 256 0C397.4 0 512 114.6 512 256z"
                fill="currentColor"
              />
            </svg>
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-dice-d20 "
              color="white"
              data-icon="dice-d20"
              data-prefix="fas"
              focusable="false"
              role="img"
              style="transform-origin: 0.5em 0.5em;"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                transform="translate(256 256)"
              >
                <g
                  transform="translate(0, 0)  scale(0.5, 0.5)  rotate(0 0 0)"
                >
                  <path
                    d="M20.04 317.3C18 317.3 16 315.8 16 313.3V150.5c0-2.351 1.91-4.012 4.001-4.012c.6882 0 1.396 .18 2.062 .5748l76.62 45.1l-75.28 122.3C22.59 316.8 21.31 317.3 20.04 317.3zM231.4 405.2l-208.2-22.06c-4.27-.4821-7.123-4.117-7.123-7.995c0-1.401 .3725-2.834 1.185-4.161L122.7 215.1L231.4 405.2zM31.1 420.1c0-2.039 1.508-4.068 3.93-4.068c.1654 0 .3351 .0095 .5089 .0291l203.6 22.31v65.66C239.1 508.6 236.2 512 232 512c-1.113 0-2.255-.2387-3.363-.7565L34.25 423.6C32.69 422.8 31.1 421.4 31.1 420.1zM33.94 117.1c-1.289-.7641-1.938-2.088-1.938-3.417c0-1.281 .6019-2.567 1.813-3.364l150.8-98.59C185.1 10.98 187.3 10.64 188.6 10.64c4.32 0 8.003 3.721 8.003 8.022c0 1.379-.3788 2.818-1.237 4.214L115.5 165.8L33.94 117.1zM146.8 175.1l95.59-168.4C245.5 2.53 250.7 0 255.1 0s10.5 2.53 13.62 7.624l95.59 168.4H146.8zM356.4 207.1l-100.4 175.7L155.6 207.1H356.4zM476.1 415.1c2.422 0 3.93 2.029 3.93 4.068c0 1.378-.6893 2.761-2.252 3.524l-194.4 87.66c-1.103 .5092-2.241 .7443-3.35 .7443c-4.2 0-7.994-3.371-7.994-7.994v-65.69l203.6-22.28C475.7 416 475.9 415.1 476.1 415.1zM494.8 370.9C495.6 372.3 496 373.7 496 375.1c0 3.872-2.841 7.499-7.128 7.98l-208.2 22.06l108.6-190.1L494.8 370.9zM316.6 22.87c-.8581-1.395-1.237-2.834-1.237-4.214c0-4.301 3.683-8.022 8.003-8.022c1.308 0 2.675 .3411 4.015 1.11l150.8 98.59c1.211 .7973 1.813 2.076 1.813 3.353c0 1.325-.6488 2.649-1.938 3.429L396.5 165.8L316.6 22.87zM491.1 146.5c2.091 0 4.001 1.661 4.001 4.012v162.8c0 2.483-2.016 4.006-4.053 4.006c-1.27 0-2.549-.5919-3.353-1.912l-75.28-122.3l76.62-45.1C490.6 146.7 491.3 146.5 491.1 146.5z"
                    fill="currentColor"
                    transform="translate(-256 -256)"
                  />
                </g>
              </g>
            </svg>
          </span>
          Hobbies & Interests
        </h3>
        <p>
          My hobbies.
        </p>
      </article>
    </div>
  </section>
  <footer
    class="footer"
  >
    <div
      class="container"
    >
      <div
        class="row gx-0 justify-content-center"
      >
        <div
          class="col-auto"
        >
          <a
            class="d-block fa-3x"
            href="https://github.com/colinhemphill"
            rel="noopener noreferrer"
            target="_blank"
          >
            <span
              class="sr-only"
            >
              Some
               on 
              GitHub
            </span>
            <span
              class="fa-layers fa-fw"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-circle "
                data-icon="circle"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M512 256C512 397.4 397.4 512 256 512C114.6 512 0 397.4 0 256C0 114.6 114.6 0 256 0C397.4 0 512 114.6 512 256z"
                  fill="currentColor"
                />
              </svg>
            </span>
          </a>
        </div>
      </div>
      <div
        class="mt-xxs"
      >
        Copyright ©
        2022
         
        Some Body
      </div>
      <div
        class="mt-xxxs"
      >
        <small>
          This résumé was generated with
           
          <a
            class="footerLink"
            href="https://nextjs.org/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Next.js
          </a>
           
          and
           
          <a
            class="footerLink"
            href="https://prismic.io/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Prismic
          </a>
          , and deployed on
           
          <a
            class="footerLink"
            href="https://vercel.com/"
            rel="noopener noreferrer"
            target="_blank"
          >
            Vercel
          </a>
          . Learn how to deploy your own resume with the
           
          <a
            class="footerLink"
            href="https://github.com/colinhemphill/nextjs-resume-generator"
            rel="noopener noreferrer"
            target="_blank"
          >
            nextjs-resume-generator
          </a>
          !
        </small>
      </div>
    </div>
  </footer>
</div>
`;
